### ISSUE TITLE
Clastic StandardEdition installation produces errors

### ISSUE THREAD

**Issue by Ozmodiar** [Clastic/Clastic#105]
The Clastic StandardEdition generates errors when installing as described in the Clastic manual:

```
... cloning master
... installing dependencies
... suggestions
... writing lock file
... generating autoload files

> ComponentInstaller\Installer::postAutoloadDump
Compiling component files
PHP Fatal error:  Call to undefined method Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installAcmeDemoBundle() in /Users/***/Projects/***/app/SymfonyStandard/RootPackageInstallSubscriber.php on line 23
PHP Stack trace:
PHP   1. {main}() /usr/local/bin/composer:0
PHP   2. require() /usr/local/bin/composer:25
PHP   3. Composer\Console\Application->run() phar:///usr/local/bin/composer/bin/composer:43
PHP   4. Symfony\Component\Console\Application->run() phar:///usr/local/bin/composer/src/Composer/Console/Application.php:82
PHP   5. Composer\Console\Application->doRun() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Application.php:126
PHP   6. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/composer/src/Composer/Console/Application.php:146
PHP   7. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Application.php:195
PHP   8. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Application.php:874
PHP   9. Composer\Command\CreateProjectCommand->execute() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:257
PHP  10. Composer\Command\CreateProjectCommand->installProject() phar:///usr/local/bin/composer/src/Composer/Command/CreateProjectCommand.php:129
PHP  11. Composer\Installer->run() phar:///usr/local/bin/composer/src/Composer/Command/CreateProjectCommand.php:171
PHP  12. Composer\EventDispatcher\EventDispatcher->dispatchScript() phar:///usr/local/bin/composer/src/Composer/Installer.php:342
PHP  13. Composer\EventDispatcher\EventDispatcher->doDispatch() phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:92
PHP  14. call_user_func() phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:152
PHP  15. SymfonyStandard\RootPackageInstallSubscriber->installAcmeDemoBundle() phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:152

Fatal error: Call to undefined method Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::installAcmeDemoBundle() in /Users/***/Projects/***/app/SymfonyStandard/RootPackageInstallSubscriber.php on line 23

Call Stack:
    0.0003     232000   1. {main}() /usr/local/bin/composer:0
    0.0157     413648   2. require('phar:///usr/local/bin/composer/bin/composer') /usr/local/bin/composer:25
    0.0557    3611368   3. Composer\Console\Application->run() phar:///usr/local/bin/composer/bin/composer:43
    0.0609    3947528   4. Symfony\Component\Console\Application->run() phar:///usr/local/bin/composer/src/Composer/Console/Application.php:82
    0.0626    4080544   5. Composer\Console\Application->doRun() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Application.php:126
    0.0642    4190944   6. Symfony\Component\Console\Application->doRun() phar:///usr/local/bin/composer/src/Composer/Console/Application.php:146
    0.0643    4191384   7. Symfony\Component\Console\Application->doRunCommand() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Application.php:195
    0.0644    4191856   8. Symfony\Component\Console\Command\Command->run() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Application.php:874
    0.0651    4197864   9. Composer\Command\CreateProjectCommand->execute() phar:///usr/local/bin/composer/vendor/symfony/console/Symfony/Component/Console/Command/Command.php:257
    0.0695    4389960  10. Composer\Command\CreateProjectCommand->installProject() phar:///usr/local/bin/composer/src/Composer/Command/CreateProjectCommand.php:129
    7.3978   51653176  11. Composer\Installer->run() phar:///usr/local/bin/composer/src/Composer/Command/CreateProjectCommand.php:171
  140.0088  173282928  12. Composer\EventDispatcher\EventDispatcher->dispatchScript() phar:///usr/local/bin/composer/src/Composer/Installer.php:342
  140.0088  173283368  13. Composer\EventDispatcher\EventDispatcher->doDispatch() phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:92
  140.0229  173539128  14. call_user_func() phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:152
  140.0229  173539192  15. SymfonyStandard\RootPackageInstallSubscriber->installAcmeDemoBundle() phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php:152

```

**Comment by NoUseFreak** [comment]
At what symfony version does this occure? Might be the recent update from 2.7 to 2.8.

**Comment by Ozmodiar** [comment]
Version 2.7.5...

**Comment by NoUseFreak** [comment]
Fixed in 448f5f1ced59ec8ed0ebf90fa3de953e81f87112
